src/js/tabs/firmware_flasher.js

```diff
@@ -4,7 +4,6 @@ const firmware_flasher = {
     targets: null,
     releaseLoader: new ReleaseLoader('https://build.betaflight.com'),
     localFirmwareLoaded: false,
-    selectedBoard: undefined,
```

Comment:
selectedBoard is used in protocols/stm32.js for validation on flashing a different target.

Reply:
Reverted :) That dependency should probably be removed.